src/components/app.js

```diff
@@ -0,0 +1,11 @@
+import Display from './display';
+import ButtonPanel from './button_panel';
+
+const App = () => (
+  <div>
+    <Display result="0" />
+    <ButtonPanel />
+  </div>
```

Comment:
Wrap children components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

Reply:
This change has not been implemented. Please do so. You can check out an example of React Fragment from the link above.